OPSEXP-2743 Run ansible-galaxy via pipenv when available (#680)
Alfresco/alfresco-build-tools: docs/README.md

### install-galaxy-deps

> Installs and cache ansible galaxy dependencies. When `~/.cache/pipenv` directory exists, installation is done on pipenv.

Reviewer: ```suggestion
Installs and cache ansible galaxy dependencies. When `~/.cache/pipenv` directory exists, installation command is prefixed with `pipenv run`.
```
